compiler/rustc_attr/src/builtin.rs

```diff
@@ -161,6 +151,20 @@ pub enum StabilityLevel {
     },
 }
 
+/// The available const-stability levels for const functions.
+/// For details see [#131349](https://github.com/rust-lang/rust/pull/131349).
```

Comment:
A PR is not a good reference for this as its description will inevitably go out of date.

Reply:
I've changed the link to be to the dev guide and moved it to the `ConstStability` struct. Though it doesn't document the internal representation, I think having the link to the description of recursive const-stability and `const_stable_indirect` on the struct is helpful.